src/core/config.ts

```diff
@@ -1,69 +1,52 @@
 import { isIOS, isAndroid } from '../utils/device-detection';
 
-import { ITitleConfig } from '../modules/ui/title/types';
-import { IFullScreenConfig } from '../modules/full-screen-manager/types';
-import { ILogoConfig } from '../modules/ui/controls/logo/types';
-import { IOverlayConfig } from '../modules/ui/overlay/types';
 import { ITextMapConfig } from '../modules/text-map/types';
-import { IScreenConfig } from '../modules/ui/screen/types';
-import { IMainUIBlockConfig } from '../modules/ui/main-ui-block/types';
 import {
   PreloadTypes,
   MediaSource,
   CrossOriginValue,
 } from '../modules/playback-engine/types';
-import { IPlayerSize } from '../modules/root-container/types';
 
 export interface IPlayerConfig {
   src?: MediaSource;
 
-  size?: IPlayerSize;
+  width?: number;
+  height?: number;
+  fillAllSpace?: boolean;
 
   videoElement?: HTMLVideoElement;
 
   preload?: PreloadTypes;
-  autoPlay?: boolean;
+  autoplay?: boolean;
   loop?: boolean;
   muted?: boolean;
   volume?: number;
-  playInline?: boolean;
+  playsinline?: boolean;
   crossOrigin?: CrossOriginValue;
+  nativeBrowserControls?: boolean;
 
-  title?: boolean | ITitleConfig;
-  loader?: boolean;
-  screen?: IScreenConfig;
-  controls?: boolean | IMainUIBlockConfig;
-  overlay?: boolean | IOverlayConfig;
-  fullScreen?: boolean | IFullScreenConfig;
-  logo?: boolean | ILogoConfig;
+  disableControlWithClickOnPlayer?: boolean;
```

Comment:
at least let's keep configuration for `title` and `overlay` and `controls`. Looks like a popular and basic use case.

Reply:
`title: string`
`shouldAlwaysShowControls: boolean`
`poster/overlay: string`